src/js/components/Button/StyledButton.js

```diff
@@ -181,6 +182,9 @@ const disabledButtonStyle = (props) => css`
         props.theme,
       )};`)}
   ${props.theme.button.disabled && props.theme.button.disabled.extend}
+  cursor: ${props.disabled
```

Comment:
Since this is already in the disabled style function, should this just be:
```
cursor: props.theme.global.control.disabled.cursor || 'default';
```

Also, can you move this above the `extend` in the line above? `extend` should always be the last thing applied.